node/src/Pos/Block/Worker.hs

```diff
@@ -97,11 +97,21 @@ blkOnNewSlot =
 -- Block creation worker
 ----------------------------------------------------------------------------
 
-blockCreator
-    :: WorkMode ssc ctx m
-    => SlotId -> SendActions m -> m ()
+blockCreator ::
+       forall ssc ctx m. WorkMode ssc ctx m
+    => SlotId
+    -> SendActions m
+    -> m ()
 blockCreator (slotId@SlotId {..}) sendActions = do
 
+    -- Print difference between tip slot and current slot.
+    -- TODO: probably not the best place, but it's not the biggest concern now.
+    tipHeader <- DB.getTipHeader @ssc
+    let tipSlot = epochOrSlotToSlot (getEpochOrSlot tipHeader)
+    let slotDiff = flattenSlotId slotId - flattenSlotId tipSlot
+    logInfo $ sformat ("Difference between current slot and tip slot is: "%int)
```

Comment:
It comes from [this commit](https://github.com/input-output-hk/cardano-sl/pull/2213/commits/a63931f7a0aa469e801457133da37a7ac30d7c3a), that we backported [here](https://github.com/input-output-hk/cardano-sl/pull/2258)